src/posts/2020-06-08-going-remote.md

```diff
@@ -0,0 +1,110 @@
+
+So, to reiterate: **don't bank on dailies** - they're a tool for communication within a project, not for isolation. **Care for asynchronous communication**, reply and encourage discussing everyday life on tools like Slack. Share gifs and videos that make you laugh. **Create moments for people to join in and goof around**. And please, if you do have a beer fountain in the office, send a few bottles to your remote coworkers, it's only common courtesy.
+
+### Decision Making
+
+We're a small company that relies on democratic values. We all have the right to an opinion. Company-wide decisions are discussed with everyone and we are free to object. But being remote, For a while I felt **more distant than ever to the decision making process**. This happened because of (1) meetings - discussed ahead, (2) offline conversations, and (3) our transition to Basecamp being rough.
+
+#### Offline conversations
+
+I'll preface this with my opinion: offline conversations are essential for a small team to act quickly and be agile. If it's not company-wide, the decision process doesn't have to go through everyone and if 2-3 people get together organically, discuss something and reach a solid conclusion, that's perfectly fine. Halting and saying "wait up, let's get this in a call and discuss it with everyone" or "let's get everyone into this" is counter-productive and doesn't allow for the fluidity you need in small teams. I'm not in the camp of "make every meeting online, even if all participants are in office" as that is a killer to productivity, at least in our team. Every year we define our yearly and quarterly OKRs and subsequently create AoRs (areas of responsibility) to make sure we achieve them. If the folks responsible for marketing need to make a quick decision (which you, a remote worker, might not care about), having a chat and discussing it offline is more than acceptable. Problems only arise when you're not communicating those decisions properly.
+
+For a while, at Subvisual, I saw this happen quite frequently and it was common that **people discussed matters offline but then fail to notify others**. The most common consequence was that someone else, not involved in the decision, would eventually make a post or lengthy argument on Slack or Basecamp and end up being followed up with a "oh, we've discussed this and it's X". Frustrations arise not because you weren't part of the decision, but because you took your time and put effort into thinking about a problem, structured your thoughts, wrote a lengthy message, only to feel dismissed.
```

Comment:
```suggestion
For a while, at Subvisual, I saw this happen quite frequently and it was common that **people discussed matters offline but then failed to notify others**. The most common consequence was that someone else, not involved in the decision, would eventually make a post or lengthy argument on Slack or Basecamp and end up being followed up with a "oh, we've discussed this and it's X". Frustrations arise not because you weren't part of the decision, but because you took your time and put effort into thinking about a problem, structured your thoughts, wrote a lengthy message, only to feel dismissed.
```